### DISCUSSION TITLE
Article URL for SEO categories

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7378] [Troubleshooting]
I have made my articles but my URL's are not how I want them. How can I make a dynamic category id folder from taxomomies that will show my full path to the article for the categories.

My Categories in my collection are taxonomies.

Am I better to add my articles as pages and categorise them via the pages ordering or am I better to add a collection and bind them to a page blueprint and then somehow categorise them via taxonomies?

Currently I have the folder structure

articles
    --index
    --show
    category
        --index
        -- show

This gives me the url:  
 /articles/the_name_of_the_article

However i'm looking for
 /articles/{name_of_the_category}/name_of_the_article

**Comment by jasonvarga** [comment]
If your `categories` field uses `max_items: 1`, it'll be saved as a string, so you can just use that field in the route.

```yaml
route: 'articles/{categories}/{slug}'
```
